addons/point_of_sale/models/pos_config.py

```diff
@@ -214,6 +215,12 @@ def _compute_current_session(self):
             pos_config.current_session_id = session and session[0].id or False
             pos_config.current_session_state = session and session[0].state or False
 
+    @api.depends('session_ids', 'session_ids.state')
+    def _compute_current_rescue_session(self):
```

Comment:
You already have a `@api.depends('session_ids', 'session_ids.state')` method in the `pos_config`. No need to create a new one, simply put your code there  (`_compute_current_session`).